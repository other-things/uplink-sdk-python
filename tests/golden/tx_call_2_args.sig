r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 68866936643352525427125634278388721948992946149422966636444761185042832544078
