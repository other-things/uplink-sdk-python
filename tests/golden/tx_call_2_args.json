{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": {
                            "contents": {
                                "decimalIntegerValue": 1,
                                "decimalPlaces": 0,
                                "tag": "Dec"
                            },
                            "tag": "NumDecimal"
                        },
                        "tag": "VNum"
                    },
                    {
                        "contents": {
                            "contents": {
                                "decimalIntegerValue": 1,
                                "decimalPlaces": 0,
                                "tag": "Dec"
                            },
                            "tag": "NumDecimal"
                        },
                        "tag": "VNum"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE02ODg2NjkzNjY0MzM1MjUyNTQyNzEyNTYzNDI3ODM4ODcyMTk0ODk5Mjk0NjE0OTQyMjk2NjYzNjQ0NDc2MTE4NTA0MjgzMjU0NDA3OA=="
}